src/app/v5_challenges/challenge.manage.user.controller.js

```diff
@@ -156,6 +154,18 @@ module.controller('v5challenge.ManageUserController', ['$scope', '$rootScope', '
             return parseInt($scope.totalCount / 100) + 1;
         };
 
-        $scope.search();
+        // get resource roles on first load and then search
+        $challengeService.v5.getResourceRoles().then(function (data) {
+            $scope.roles = $scope.roles.concat(data);
+            $scope.filterCriteria.roleId = $scope.roles.filter(function(role) {
+                return role.name === DEFAULT_ROLE_FILTER_NAME;
+            }).map(function (role) {
+                return role.id;
+            })[0];
```

Comment:
It looks like we are searching here, not filtering, so instead of `filter` we can use a `find` method from lodash:
```js
$scope.filterCriteria.roleId = _.find($scope.roles, { name: DEFAULT_ROLE_FILTER_NAME }).id
```

Reply:
ended up removing this filter as I changed the logic a bit to update the url